History.markdown

```diff
@@ -1,4 +1,4 @@
-## HEAD
+## 3.8.2 / 2018-05-18
```

Comment:
Maybe I should have updated this date to today 🤷‍♂️

Reply:
There is no need, PR was created yesterday.